internal/cri/config/config.go

```diff
@@ -682,6 +694,68 @@ func ValidateServerConfig(ctx context.Context, c *ServerConfig) ([]deprecation.W
 	return warnings, nil
 }
 
+func (config *ContainerdConfig) loadRuntime(handler string) (Runtime, error) {
+	r, ok := config.Runtimes[handler]
+	if ok {
+		return r, nil
+	}
+
+	p := filepath.Join(defaults.DefaultConfigDir, "cri.runtimes", handler, "config.toml")
```

Comment:
maybe good to add a config path like `RuntimesConfigPath` to allow users to set the root folder themselves.

also I think it'd be good if we can have a file watcher and update configs on modifications, otherwise it feels a bit confusing if one can add a new runtime w/o restart, but cannot modify an existing one w/o restart?

Reply:
You can modify one without needing to restart.

Reply:
Added config.